### ISSUE TITLE
No tooltip on legend items when accessibility module is attached

### ISSUE THREAD

**Issue by violetVo** [highcharts/highcharts#13765]
#### Expected behaviour
The legend items should have tooltip when the value is truncated regardless the accessibility module is attached, or not.

#### Actual behaviour
When accessibility module is attached there is no tooltip for the truncated legend items and there is no option to see the actual value.

#### Live demo with steps to reproduce
https://codepen.io/Voronetzky/pen/ExPWGVR

See with and whithout accesibility module:
![image](https://user-images.githubusercontent.com/17854021/85575278-18e45200-b640-11ea-9071-2ab21a2492ef.png)

#### Product version
Highcharts 8.1.1

#### Affected browser(s)
Checked in Chrome

**Comment by pawelfus** [comment]
Thank you for reporting the issue @violetVo 

It seems that accessibility's module button is covering text that has title attributes set. FYI @oysteinmoseng

**Comment by oysteinmoseng** [comment]
An easy workaround is to reduce the size of the overlay button: https://codepen.io/oysteinmoseng/pen/jOWGwPx. The only downside here is really for voice input users where there might be a focus border around the button, but it is a marginal difference with most software. Marginal enough that we could consider always doing this internally.
